paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExternalPaymentMethodHandler.kt

```diff
@@ -0,0 +1,7 @@
+package com.stripe.android.paymentsheet
+
+internal object ExternalPaymentMethodHandler {
+    fun confirm(onComplete: (PaymentSheetResult) -> Unit) {
```

Comment:
This is not what the function will actually accept as params once it's doing something more realistic, but this works well now for an early exit